### DISCUSSION TITLE
Can default property value be restored through CSS?

### DISCUSSION THREAD

**Discussion by olx24** [STRINGLABITSOLUTIONS/Bhojon#146] [Q&A] [answered]
Help me to get one

**Comment by montasim** [comment, marked as answer]
In CSS, you cannot revert back to old values due to lack of default values. The property can be re- declared to get the default property.

**Comment by Preetiraj3697** [comment]
In short, there's no easy way to restore to default values to whatever a browser uses . The closest option is to use the 'initial' property value, which will restore it to the default CSS values, rather than the browser's default styles.
